djstripe/models.py

```diff
@@ -497,10 +497,11 @@ def sync_current_subscription(self, cu=None):
             return sub_obj
 
     def update_plan_quantity(self, quantity, charge_immediately=False):
+        sub = self.stripe_customer.subscription
```

Comment:
We're in the process of moving away from using ``sub``, as it isn't very descriptive. I think this would be better suited as ``stripe_subscription``